test/e2e/util.go

```diff
@@ -827,11 +827,14 @@ func waitForRCPodsRunning(c *client.Client, ns, rcName string) error {
 // Delete a Replication Controller and all pods it spawned
 func DeleteRC(c *client.Client, ns, name string) error {
 	By(fmt.Sprintf("Deleting replication controller %s in namespace %s", name, ns))
-	reaper, err := kubectl.ReaperFor("ReplicationController", c)
+	reaper, err := kubectl.ReaperForReplicationController(c, 10*time.Minute)
```

Comment:
It's not enough, because pollInterval and timeout are private.
However, I'm fine with making the public if you think it's better approach.
WDYT?